runestone/activecode/js/activecode_brython.js

```diff
@@ -0,0 +1,63 @@
+import { ActiveCode } from "./activecode.js";
+
+export default class BrythonActiveCode extends ActiveCode {
+    constructor(opts) {
+        super(opts);
+        opts.alignVertical = true;
+        this.python3_interpreter = $(orig).data("python3_interpreter");
+        $(this.runButton).text("Render");
+        this.editor.setValue(this.code);
+    }
+
+    async runProg() {
+        var prog = await this.buildProg(true);
+        let saveCode = "True";
+        this.saveCode = await this.manage_scrubber(saveCode);
+        $(this.output).text("");
+        if (!this.alignVertical) {
+            $(this.codeDiv).switchClass("col-md-12", "col-md-6", {
+                duration: 500,
+                queue: false,
+            });
+        }
+        $(this.outDiv).show({ duration: 700, queue: false });
+        prog = `
+        <html>
+            <head>
+                <script type='text/javascript' src='https://cdn.jsdelivr.net/npm/brython@3.9.0/brython.min.js'></script>
+            </head>
+            <body onload='brython()'>
+                <script type='text/python'>` + prog + 
```

Comment:
I'm amazed at how -- seemingly easy -- it was to add Brython support!
Since you are using interpolated strings (`) you should eliminate the + and use ${prog} to insert prog into the string.

Reply:
Thank you for the suggestion. I will commit it right away! :)